Forge/Server/Data/LiteDbCharacterService.cs

```diff
@@ -43,6 +43,26 @@ public bool Update(CharacterModel character)
                 .Update(character);
         }
 
+        public bool DeleteOne(Guid id)
+        {
+            var character = _liteDb.GetCollection<CharacterModel>("Character")
```

Comment:
Please use the "FindOne" repository method instead of directly accessing the collection.